src/components/editor/Editor.tsx

```diff
@@ -135,6 +137,11 @@ class Editor extends React.Component<
     if (!this.state.annotationLinkTitle) {
       this.setState({ annotationLinkTitleError: 'Musisz podać tytuł źródła, jeśli chcesz dodać przypis!' });
       return false;
+    } else if (this.state.annotationLinkTitle.length > Editor.linkTitleMaxLength) {
+      this.setState({ annotationLinkTitleError:
+          `Skróć tytuł źródła z ${this.state.annotationLinkTitle.length} do ${Editor.linkTitleMaxLength} znaków!`,
```

Comment:
I guess with the current function size it might be a little bit more elegant with something like
```js
const { 
  annotationLinkTitle, 
  annotationLinkTitleError,
  (...)
} = this.state;
```
But that's just a thought.

Reply:
Just a thought, but good one :P